### PR TITLE
JetConfig#toString() impl

### PR DESCRIPTION
**Pull request by xitep**
because this `... ogConfig{factoryClassName='null', enabled=false, properties={}}, jetConfig=com.hazelcast.jet.config.JetConfig@bebd958a, deviceConfigs ...` isn't revealing much.

### CODE REVIEW

**Comment by TomaszGaweda** [comment]
run-lab-run

**Review by TomaszGaweda — APPROVED** [review]

**Comment by TomaszGaweda** [comment]
Thanks for your contribution @xitep! I think it may be helpful for debugging, so I've approved. I've asked one of my collegues to check it as well, maybe he will have some objections ;)

File: hazelcast/src/main/java/com/hazelcast/jet/config/JetConfig.java [hunk 1]
```diff
@@ -378,4 +378,20 @@ public int hashCode() {
         result = 31 * result + (resourceUploadEnabled ? 1 : 0);
         return result;
     }
+
+    @Override
+    public String toString() {
+        return "JetConfig{"
+                + "instanceConfig=" + instanceConfig
```
Comment (k-jamroz) [inline hazelcast/src/main/java/com/hazelcast/jet/config/JetConfig.java:385]: there is no point in logging instanceConfig, it is deprecated and always DelegatingInstanceConfig

**Comment by TomaszGaweda** [comment]
Can you sign the CLA @xitep? It's required due to legal reasons to have your confirmation, that the code can be used :)

**Comment by TomaszGaweda** [comment]
run-lab-run

**Comment by TomaszGaweda** [comment]
run-lab-run

**Pull request merged**